src/commands/transaction/accountaddressrestriction.ts

```diff
@@ -0,0 +1,136 @@
+import { command, metadata, option } from 'clime';
+import {
+    Account,
+    AccountRestrictionModification,
+    AccountRestrictionModificationAction,
+    AccountRestrictionTransaction,
+    AccountRestrictionType,
+    Address,
+    Deadline,
+    TransactionHttp,
+    UInt64,
+} from 'nem2-sdk';
+import { OptionsResolver } from '../../options-resolver';
+import { ProfileCommand, ProfileOptions } from '../../profile.command';
+import { ModificationActionValidator } from '../../validators/modificationAction.validator';
+import { RestrictionTypeValidator } from '../../validators/restrictionType.validator';
+
+export class CommandOptions extends ProfileOptions {
+    @option({
+        flag: 'p',
+        description: '(Optional) Select between your profiles, by providing a profile (blank means default profile)',
+    })
+    profile: string;
+
+    @option({
+        flag: 't',
+        description: 'restriction type (allow / block)',
```

Comment:
We could use the binary validator to make it consistent across commands (1 allow, 0 block)